geoips/plugins/modules/procflows/config_based.py

```diff
@@ -37,7 +38,8 @@
         write_stats_to_database,
     )
 except ImportError:
-    print("Please install geoips_db package if required")
+    pass
+    # print("Please install geoips_db package if required")
```

Comment:
Should this just be deleted rather than commented out?

Reply:
Done.